pwa/src/services/auth.tsx

```diff
@@ -0,0 +1,36 @@
+import { navigate } from "gatsby-link";
+import jwtDecode, { JwtPayload } from "jwt-decode";
+
+export const isBrowser = () => typeof window !== "undefined";
+
+export const getUser = () =>
+  isBrowser() && window.sessionStorage.getItem("user") ? JSON.parse(window.sessionStorage.getItem("user")) : {};
+
+export const setUser = (user) => window.sessionStorage.setItem("user", JSON.stringify(user));
```

Comment:
We're treating the `user` parameter as `any`, but we're not explicitly typing it as such. Meaning that we are _implicitly_ typing it, which is not allowed.

The easiest fix for these errors is to _explicitly_ type it as `any`, like so:

```
export const setUser = (user: any) => ...
```

Explicitly typing something to `any` is allowed, when we do not have a type available. 

However, in this case, I commented on the `User` type before, I think we might be able to actually type this (maybe even from Gatsby itself).